Encapsulation Exercises/Digo-encapsulation-Project/src/main/java/encapsulation/Main.java

```diff
@@ -0,0 +1,72 @@
+package main.java.encapsulation;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        double width = 0;
+        double height = 0;
+        double depth = 0;
+
+        boolean validInput = false;
+        while (!validInput) {
+            try {
+                System.out.println("Enter width of the box:");
+                width = Double.parseDouble(scanner.nextLine());
+                if (width <= 0) {
+                    System.out.println("Width must be a positive number.");
+                }
+                validInput = true;
+            } catch (IllegalArgumentException e) {
+                System.out.println("Invalid Input. Use Number!");
+            }
+        }
+
+        validInput = false;
+        while (!validInput) {
```

Comment:
use only one loop to facilitate all the inputs. use continue or break statements when some validations are needed